WebApi/Controllers/HomeController.cs

```diff
@@ -19,5 +22,16 @@ public IHttpActionResult GetTest()
 		{
 			return Json("Hello World ABCDEFG");
 		}
+
+		public OkNegotiatedContentResult<string> Hello()
```

Comment:
Add a comment here. The plan is to keep using `Json(_object)` until I can figure out what this does.